nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/pom.xml

```diff
@@ -116,5 +116,10 @@
             <version>${nifi.groovy.version}</version>
             <scope>test</scope>
         </dependency>
-    </dependencies>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20180813</version>
```

Comment:
I believe we don't want to use this library because of the licensing.
https://jira.apache.org/jira/browse/LEGAL-245
to be confirmed though - I believe there are accepted alternatives

Reply:
But it's for GOOD 👼 
Ok! I'll look for an alternative. Thanks.

Reply:
@pvillard31 evil json replaced.